<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-4N84MDJ" failures="1" tests="1" name="Order Managment Validation" time="40.938" errors="0" timestamp="2024-04-15T09:16:42 IST">
  <testcase classname="retailer.tests.ordermanagmentvalidation" name="rejectorder" time="29.642">
    <failure type="java.lang.StringIndexOutOfBoundsException" message="begin 1, end 0, length 0">
      <![CDATA[java.lang.StringIndexOutOfBoundsException: begin 1, end 0, length 0
at java.base/java.lang.String.checkBoundsBeginEnd(String.java:4604)
at java.base/java.lang.String.substring(String.java:2707)
at java.base/java.lang.String.substring(String.java:2680)
at retailer.pageobject.OrderHistory.getrejectedid(OrderHistory.java:45)
at retailer.tests.ordermanagmentvalidation.rejectorder(ordermanagmentvalidation.java:37)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- rejectorder -->
</testsuite> <!-- Order Managment Validation -->
